scene/3d/remote_transform_3d.cpp

```diff
@@ -140,7 +141,14 @@ NodePath RemoteTransform3D::get_remote_node() const {
 }
 
 void RemoteTransform3D::set_use_global_coordinates(const bool p_enable) {
+	if (use_global_coordinates == p_enable) {
+		return;
+	}
+
 	use_global_coordinates = p_enable;
+
+	set_notify_transform(use_global_coordinates);
+	set_notify_local_transform(!use_global_coordinates);
```

Comment:
I think `_update_remote()` is also needed in here, no ? :thinking: (for 2D it's in there)
```suggestion
	set_notify_transform(use_global_coordinates);
	set_notify_local_transform(!use_global_coordinates);
	_update_remote();
```

Reply:
Yes, it makes sense.